arangod/Replication/DatabaseInitialSyncer.cpp

```diff
@@ -187,7 +187,21 @@ arangodb::Result fetchRevisions(
   std::string path = arangodb::replutils::ReplicationUrl + "/" +
                      RestReplicationHandler::Revisions + "/" +
                      RestReplicationHandler::Documents;
-  auto headers = arangodb::replutils::createHeaders();
```

Comment:
Is it intentional that this is removed?

Reply:
Ah, I see, this local variable was never used. LOL.

Reply:
Exactly.